### PR TITLE
Fixes part of #40 & #42: Highfi Submitted answer 

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

Fixes part of #40
Fixes part of #42 

This PR focuses on complete Highfi of `submittted_answer_item` xml file.

Currently only the outer margins have been done and inner UI is pending.

Waiting for inner UI from Chantel.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by rt4914** [comment]
@mschanteltc In the component guide can you add the UI (internal and not the external margins) for `submitted-anwer`. 
You can create a [page](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/740f3983-c167-4ce4-a3e1-a72b387cefc2/specs/) similar to this for exploration as well as question player.

`Submitted-answer` (white box in each screenshot) can be of following types:

      
   

Basically,
- Submitted Answer - Text - in exploration player
- Submitted Answer - Text - in question player (correct with tick as well  as incorrect without tick)
- Submitted Answer - List - in exploration player
- Submitted Answer - List - in question player (correct with tick as well  as incorrect without tick)

**Comment by rt4914** [comment]
@mschanteltc PTAL whenever possible

**Comment by mschanteltc** [comment]
> @mschanteltc PTAL whenever possible

Hi Rajat. I created mocks for these in mobile portrait view for your review before I create mocks for the other layouts. PTAL:

- [Exploration Player](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/92b8180c-733d-46f2-b2d6-5c07f7c56680/)
- [Practice Player (Incorrect Answer)](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/2df353ea-dcf7-4dc9-9008-e43ab14378f1/)
- [Practice Player (Correct Answer)](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/e30f0d1b-5618-4f02-b3b7-58fa92e9a14c/)

I also edited the Practice Player layout. The [old](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/efbce213-79cc-4dfd-8efd-97a9f3265d0b/) one did not account for the checkmark, but the [edited](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/4a64cf26-7a15-4ead-84de-7bb04864a0ec/) version does. There will always be a minimum of 32px right padding to the User's Answer. Let me know if you have any questions.

**Comment by rt4914** [comment]
Solved in #3273
